src/jaxsot/core/testdata.py

```diff
@@ -34,3 +34,41 @@ def static_singleband(nside):
     W = jnp.array(WI * WV)
     lc = gen_lightcurve(W, mmap, 0.1)
     return lc, W
+
+def static_multiband():
+    """generate test data for a multi-band observation of a static geography
+    
+    Returns:
+        test light curve, weight matrix
+    
+    """
+
+    mmap, Ainit, Xinit = multibandmap(show=False)
+    npix = mmap.shape[0]
+    nside = hp.npix2nside(npix)
+
+    inc = 0.0
```

Comment:
Can you move these parameters to function arguments w/ default values?
like 
`static_multiband(inc=0.0, ... )`